roles/st2/tasks/datastore.yaml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Add the st2 system service user to the service group
+  become: yes
+  user:
+    user: "{{ st2_system_service_user }}"
+    groups: "{{ st2_system_service_group }}"
```

Comment:
Creation of `st2` user and group is already done in packaging so this block is not needed.

Reply:
Then maybe there was a bug, because the st2 user was not a member of the st2 group, so I added this to add the user to the group. The user already existed. Maybe this issue has been addressed elsewhere?

Reply:
This works for me on Ubuntu16:
```
groups st2
st2 : st2 st2packs
```
`st2` user was associated with `st2` group. I'm running latest s2 2.4

Do you see any diff on EL/CentOS systems? Must be a bug then.

Reply:
Reading the rpm spec, it should have created the group. I don't know what I saw as it was a while ago. I'll just delete this for now, and if it pops up again, I'll file a bug against the st2-packages.